tests/unit/test_pep621.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of CycloneDX Python
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OWASP Foundation. All Rights Reserved.
+
+import os
+import tempfile
+import unittest
+
+from cyclonedx.factory.license import LicenseFactory
+from cyclonedx.model.license import DisjunctiveLicense, LicenseAcknowledgement
+
+from cyclonedx_py._internal.utils.pep621 import project2licenses
+
+
+class TestPEP621(unittest.TestCase):
+    def setUp(self):
```

Comment:
the setup seams to have not much value. put the constructors and such into the nest functions